specification/monitor/resource-manager/Microsoft.Insights/preview/2020-05-01-preview/scheduledQueryRule_API.json

```diff
@@ -333,6 +333,10 @@
           "type": "string",
           "description": "The description of the scheduled query rule."
         },
+        "displayName": {
```

Comment:
This is a fix to s360 violation :
https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=2EAB7F16-67CA-44F1-8877-39B8404F881F